perfscale_regression_ci/content/ovn_metrics_network_policy.yaml

```diff
@@ -0,0 +1,14 @@
+apiVersion: networking.k8s.io/v1
+kind: NetworkPolicy
```

Comment:
this looks exactly the same as [file](https://github.com/openshift/svt/blob/master/perfscale_regression_ci/content/ovn-allow_default_network_policy.yaml) that is already in the master branch, can you reuse that one and get rid of this one?